app/views/chugs/_podium_list.html.erb

```diff
@@ -0,0 +1,91 @@
+<ul role="list" class="divide-y divide-gray-200">
+  <% if chugs.count >= 3 %>
+    <% chugs.each_with_index do |chug, index|  %>
+      <% if index == 0 %>
+        <li class="flex items-center justify-center">
+          <div class="podium py-4">
+            <img class="medal gold" src="/img/medal_gold.png" alt="Gold medal">
+            <%= link_to chug.user, class: "self-center" do %>
+              <% gravatar_for chug.user, class: "gravatar gold-gravatar rounded-full" %>
+            <% end %>
+            <div class="flex items-center justify-center">
+              <p class="text-center gold-text" style="width: 100px; line-height: 1.1rem"><strong><%= chug.secs %>.<%= sprintf("%02d", chug.milis) %></strong>s
+              <br> <i style="font-size: 0.85rem;"><%= chug.user.name %></i></p>
+            </div>
+      <% elsif index == 1 %>
+            <img class="medal silver" src="/img/medal_silver.png" alt="Gold medal">
+            <%= link_to chug.user, class: "self-center" do %>
+              <% gravatar_for chug.user, class: "gravatar silver-gravatar rounded-full" %>
+            <% end %>
+            <div class="flex items-center justify-center">
+              <p class="text-center silver-text" style="width: 100px; line-height: 1.1rem"><strong><%= chug.secs %>.<%= sprintf("%02d", chug.milis) %></strong>s
+                <br> <i style="font-size: 0.85rem;"><%= chug.user.name %></i></p>
+            </div>
+      <% elsif index == 2 %>
+            <img class="medal bronze" src="/img/medal_bronze.png" alt="Gold medal">
+            <%= link_to chug.user, class: "self-center" do %>
+              <% gravatar_for chug.user, class: "gravatar bronze-gravatar rounded-full" %>
+            <% end %>
+            <div class="flex items-center justify-center">
+              <p class="text-center bronze-text" style="width: 100px; line-height: 1.1rem"><strong><%= chug.secs %>.<%= sprintf("%02d", chug.milis) %></strong>s
+                <br> <i style="font-size: 0.85rem;"><%= chug.user.name %></i></p>
+            </div>
+          </div>
+        </li>
+      <% else %>
+        <%= render 'chugs/chug', chug: chug %>
+      <% end %>
+    <% end %>
+  <% else %>
+    <%= render chugs %>
+    <div class="py-4 pb-4">
+      <i>
+        Er moeten minimaal 3 adtjes zijn om het scorebord te kunnen zien.
+      </i>
+    </div>
+  <% end %>
+</ul>
+
+<style>
```

Comment:
De style doen we eigenlijk nooit inline maar kun je in frontend/stylesheets kwijt. We gebruiken op het moment geen custom css maar alleen dingen uit tailwind. Zou heel mooi zijn als je die classes kan gebruiken in plaats van eigen classes definiëren. Als wel custom CSS nodig is graag naar een eigen file verplaatsen.

Reply:
Meeste hiervan is gefixt, maar precieze offset is moeilijk te definen met tailwind. Zou mogelijk zijn als we jit gebruiken, maar dat zie ik nog nergens voorkomen. Is dit prima als dit dus in een apart css file terugkomt?